### ISSUE TITLE
Typescript type definitions missing in v6.0.1

### ISSUE THREAD

**Issue by OpportunityLiu** [nfroidure/svg-pathdata#59]
## Issue

I'm a gentledev i:
- [x] fully read the README recently
- [x] searched for existing issues
- [x] checked i'm up to date with the latest version of the project

### Expected behavior
https://unpkg.com/browse/svg-pathdata@6.0.0/lib/
### Actual behavior
https://unpkg.com/browse/svg-pathdata@6.0.1/lib/
### Debugging informations
- `node -v` result:
```
<paste here>
```

- `npm -v` result:
```
<paste here>
```
If the result is lower than 6.9.5, there is
 poor chances i even have a look to it. Please,
 use the last [NodeJS LTS version](https://nodejs.org/en/).

**Comment by leon** [comment]
Same problem.
using 6.0.1 broke the library.
```
import { SVGPathData } from 'svg-pathdata'
```
tells me `Module '"svg-pathdata"' has no exported member 'SVGPathData'.`

**Comment by pioug** [comment]
Can someone give a reproducible example? or some pointers for any solution

**Comment by pioug** [comment]
Released v6.0.2, it should fix the initial problem https://unpkg.com/browse/svg-pathdata@6.0.2/lib/

@leon Not sure if it's was the same problem, create another issue *with a repro example* if it persists
